tdrs-backend/tdpservice/scheduling/db_backup.py

```diff
@@ -140,32 +149,32 @@ def restore_database(file_name, postgres_client, database_uri, system_user):
             change_message=msg
         )
         logger.info(msg)
-    except Exception as e:
-        logger.error(f"Caught exception while creating the database. Exception: {e}.")
-        return False
 
-    # write .pgpass
-    with open('/home/vcap/.pgpass', 'w') as f:
-        f.write(DATABASE_HOST+":"+DATABASE_PORT+":"+DATABASE_DB_NAME+":"+DATABASE_USERNAME+":"+DATABASE_PASSWORD)
-    os.environ['PGPASSFILE'] = '/home/vcap/.pgpass'
-    os.system('chmod 0600 /home/vcap/.pgpass')
+        # write .pgpass
+        with open('/home/vcap/.pgpass', 'w') as f:
+            f.write(DATABASE_HOST+":"+DATABASE_PORT+":"+DATABASE_DB_NAME+":"+DATABASE_USERNAME+":"+DATABASE_PASSWORD)
+        os.environ['PGPASSFILE'] = '/home/vcap/.pgpass'
+        os.system('chmod 0600 /home/vcap/.pgpass')
 
-    logger.info("Begining database restoration.")
-    cmd = (postgres_client + "pg_restore" + " -p " + DATABASE_PORT + " -h " +
-           DATABASE_HOST + " -U " + DATABASE_USERNAME + " -d " + DATABASE_DB_NAME + " " + file_name)
-    logger.info(f"Executing restore command: {cmd}")
-    os.system(cmd)
-    msg = "Completed database restoration."
-    LogEntry.objects.log_action(
-        user_id=system_user.pk,
-        content_type_id=content_type.pk,
-        object_id=None,
-        object_repr="Executed Database restore",
-        action_flag=ADDITION,
-        change_message=msg
-    )
-    logger.info(msg)
-    return True
+        logger.info("Begining database restoration.")
+        cmd = (postgres_client + "pg_restore" + " -p " + DATABASE_PORT + " -h " +
+               DATABASE_HOST + " -U " + DATABASE_USERNAME + " -d " + DATABASE_DB_NAME + " " + file_name)
+        logger.info(f"Executing restore command: {cmd}")
+        os.system(cmd)
+        msg = "Completed database restoration."
+        LogEntry.objects.log_action(
+            user_id=system_user.pk,
+            content_type_id=content_type.pk,
+            object_id=None,
+            object_repr="Executed Database restore",
+            action_flag=ADDITION,
+            change_message=msg
+        )
+        logger.info(msg)
+        return True
+    except Exception as e:
+        logger.error(f"Caught exception while creating the database. Exception: {e}.")
```

Comment:
```suggestion
        logger.error(f"Caught exception while restoring the database. Exception: {e}.")
```

I know it's not really your scope, but after reviewing this expanded try/except, felt this was more apt description.